### PR TITLE
[EPMEDU-3540]: Hide feeding approval flow on prod

### PR DESCRIPTION
**Pull request by Fileek**
[Ticket ref](https://jira.epam.com/jira/browse/EPMEDU-3540)
Hid buttons that lead to the approval flow.

P. S. This should be reverted as soon as the approval flow is ready ([ticket](https://jira.epam.com/jira/browse/EPMEDU-3539)).

### CODE REVIEW

File: shared/feature/feedings/src/main/java/com/epmedu/animeal/feedings/presentation/viewmodel/handlers/FeedingsButtonHandlerImpl.kt [hunk 1]
```diff
@@ -10,25 +11,34 @@ import com.epmedu.animeal.users.domain.model.UserGroup
...
-        return combine(
-            getIsNewFeedingPendingUseCase(),
-            flow { emit(getUserGroupUseCase()) }
-        ) { isNewFeedingPending, userGroupResult ->
-            when {
-                userGroupResult.isEligibleForFeedingsButton() -> {
-                    if (isNewFeedingPending) Pulsating else Static
-                }
+        return when {
+            buildConfigProvider.isProdFlavor -> {
```
Comment (ozkarram) [inline shared/feature/feedings/src/main/java/com/epmedu/animeal/feedings/presentation/viewmodel/handlers/FeedingsButtonHandlerImpl.kt:25]: This way feedings button would be visible at dev/test environment only

**Review by ozkarram — CHANGES_REQUESTED** [review]

File: shared/feature/feedings/src/main/java/com/epmedu/animeal/feedings/presentation/viewmodel/handlers/FeedingsButtonHandlerImpl.kt (hunk 1 above)
Comment (Fileek) [reply shared/feature/feedings/src/main/java/com/epmedu/animeal/feedings/presentation/viewmodel/handlers/FeedingsButtonHandlerImpl.kt:25]: Yes, we do not need it in prod until it is fully ready

Comment (ozkarram) [reply shared/feature/feedings/src/main/java/com/epmedu/animeal/feedings/presentation/viewmodel/handlers/FeedingsButtonHandlerImpl.kt:25]: Pls add a tracking ticket for this and add it as comment, we will need to know what changes are necessary in order to make it work on prod env.

**Review by ozkarram — CHANGES_REQUESTED** [review]
The new ticket could be part of prod delivery or some of our tickets, but we need to keep track of these changes to avoid confusion in the future.

File: shared/feature/feedings/src/main/java/com/epmedu/animeal/feedings/presentation/viewmodel/handlers/FeedingsButtonHandlerImpl.kt (hunk 1 above)
Comment (Fileek) [reply shared/feature/feedings/src/main/java/com/epmedu/animeal/feedings/presentation/viewmodel/handlers/FeedingsButtonHandlerImpl.kt:25]: Done

**Review by ozkarram — APPROVED** [review]

**Pull request merged**